### PR TITLE
OCPBUGS-1251: Add admin-gate ack-4.11-kube-1.25-api-removals-in-4.12

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Reverting the last revert as we want to add this just before 4.12 feature freeze. This reverts commit f27e7b57c1a2bee0a4670887c233899c32126491.

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: install/0000_00_cluster-version-operator_01_admingate_configmap.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+  ack-4.11-kube-1.25-api-removals-in-4.12: |
```
Comment (wking) [inline install/0000_00_cluster-version-operator_01_admingate_configmap.yaml:4]: @bergerhoffer, in [OSDOCS-3571][1], you'd asked to be tagged in this pull request.

[1]: https://issues.redhat.com/browse/OSDOCS-3571

**Review by wking — APPROVED** [review]
/lgtm

File: install/0000_00_cluster-version-operator_01_admingate_configmap.yaml (hunk 1 above)
Comment (bergerhoffer) [reply install/0000_00_cluster-version-operator_01_admingate_configmap.yaml:4]: Excellent thank you @wking! I'll make a note of this PR for when we update the KBase and 4.12 product docs w/ the actual ack.

**Review by wking — APPROVED** [review]
/lgtm

**Review by wking — APPROVED** [review]
/lgtm

**Comment by evakhoni** [comment]
pre merge verified in https://issues.redhat.com/browse/OCPBUGS-1251?focusedCommentId=20958917&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-20958917
/label qe-approved

File: install/0000_00_cluster-version-operator_01_admingate_configmap.yaml (hunk 1 above)
Comment (bergerhoffer) [reply install/0000_00_cluster-version-operator_01_admingate_configmap.yaml]: @wking Do you know if this is live in the latest 4.11 z-stream yet? Just wondering if we can update the kbase articles to include this info yet.

**Pull request merged**